pkg/csconfig/api.go

```diff
@@ -230,6 +258,14 @@ func (c *Config) LoadAPIServer() error {
 	if c.API.Server.OnlineClient == nil || c.API.Server.OnlineClient.Credentials == nil {
 		log.Printf("push and pull to Central API disabled")
 	}
+
+	log.Printf("Config load, c.API.CTI : %p", c.API.CTI)
```

Comment:
Shouldn't the line be inside the nil catch as most people start will get nil at start?